SelectorCSS.java

```diff
@@ -0,0 +1,65 @@
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+public class SelectorCSS {
+     WebDriver driver;
+
+     @BeforeClass
+     public void setUp() {
+          WebDriverManager.chromedriver().setup();
+          driver = new ChromeDriver();
+          driver.manage().window().maximize();
+
+     }
+
+     @BeforeMethod
+
+     public void navigate() {
+          driver.get("http://the-internet.herokuapp.com/checkboxes");
+
+     }
+
+     @AfterClass
+
+     public void tearDown() {
+          driver.quit();
+
+     }
+
+     @Test
+     public void CheckboxesTest() throws InterruptedException {
+
+          WebElement CheckBox1 = driver.findElement(By.cssSelector("#checkboxes > input[type=checkbox]:nth-child(1)"));
```

Comment:
It's better to use List<WebElements> if there're some similar elements on the page like these checkboxes and then acces them by index: checkboxes.get(0) / get(1)